### ISSUE TITLE
Undefined names & missing pictures

### ISSUE THREAD

**Issue by biorisk2** [ztizzlegaming/CSGOWinBig#19]
I've followed the steps quite well, the pots working & what not. However anything steam related doesn't show up 

https://i.gyazo.com/4bc2e973b615310f910405ab7a866573.png

**Comment by RatexIndex** [comment]
Yes we all have this Problem i think thats a update problem i hope that is fixed next time ;)

**Comment by biorisk2** [comment]
I managed to fix the issue.

**Comment by TheAnthonyNL** [comment]
@biorisk2 well share it with us :).
